stylelint-polaris/tsconfig.json

```diff
@@ -1,11 +0,0 @@
-{
-  "extends": "@shopify/typescript-configs/node-library.json",
-  "compilerOptions": {
-    "baseUrl": ".",
-    "outDir": "dist",
-    "allowJs": true,
-    "checkJs": true,
-    "skipLibCheck": true
-  },
-  "exclude": ["node_modules", "dist"]
-}
```

Comment:
Please keep JSDoc type validation in this library.

Reply:
I think I remember that one! Pretty sure I just changed `const` to `let`. I can help look into it if that doesn't resolve the issue (Don't want to block this work).

Reply:
Oooh eslint flipped that back to `const` in #5430. Guess we'll need an eslint disable line to shut that up